src/main/java/no/ndla/taxonomy/service/DomainEntityHelperServiceImpl.java

```diff
@@ -319,49 +298,33 @@ private ResourceType ensureResourceTypesExists(ResourceType resourceType) {
     }
 
     protected void mergeMetadata(EntityWithMetadata present, Metadata metadata, URI publicId, boolean cleanUp) {
-        if (present == null) {
-            // Make sure custom fields exists
-            metadata.getCustomFieldValues().forEach(customFieldValue -> customFieldService.setCustomField(null,
-                    customFieldValue.getCustomField().getKey(), null));
-            return;
-        }
         Metadata presentMetadata = present.getMetadata();
```

Comment:
Fikser testene hos meg.
```suggestion
        if (present == null) {
            return;
        }
        Metadata presentMetadata = present.getMetadata();
```

Reply:
Dvs tre tester feila også med denne endringa.